demos/supabase-anonymous-auth/lib/powersync.dart

```diff
@@ -137,8 +136,13 @@ String? getUserId() {
 }
 
 Future<String> getDatabasePath() async {
-  final dir = kIsWeb ? Directory('/') : await getApplicationSupportDirectory();
-  return join(dir.path, 'powersync-demo.db');
+  const dbFilename = 'powersync-demo.db';
+  // getApplicationSupportDirectory is not supported on Web
+  if (kIsWeb) {
```

Comment:
what is kIsWeb? Can this be named more clearly?

Reply:
This is a standard import from Flutter. The naming comes from them.